modules/serving-transport-encryption-choice-of-a-certificate-issuer.adoc

```diff
@@ -0,0 +1,49 @@
+:_mod-docs-content-type: CONCEPT
+[id="serving-transport-encryption-choice-of-a-certificate-issuer_{context}"]
+= Choice of a certificate issuer
+
+[NOTE]
+====
+Issuers refer to cert-manager `Issuers` and `ClusterIssuers`. They represent certificate authorities (CAs) that can generate signed certificates by honoring certificate signing requests.
+====
+
+To identify your certificate issuer, use `cert-manager` documentation, which has examples for the following:
+
+* A custom CA stored in a K8s secret
+* HTTP-01 challenges, e.g. Let’s encrypt
+* DNS-01 challenges
+* Self-signed issuers
+
+[NOTE]
+====
+Not all issuer types work for each Knative Serving encryption feature.
+====
+
+Depending on the encryption part you use, {ServerlessProductName} requires your certificate issuer to be able to sign certain certificates.
```

Comment:
```suggestion
Depending on the encryption features that you use, {ServerlessProductName} requires your certificate issuer to be able to sign certain certificates. To identify your certificate issuer, refer to the `cert-manager` documentation, which contains examples for the following:

* A custom CA stored in a Kubernetes secret
* HTTP-01 challenges
* DNS-01 challenges
* Self-signed issuers

== Add heading

Not all issuer types work for each Knative Serving encryption feature.

```
Instead of having the "Not all issuer types work for each Knative Serving encryption feature." part as a note here, I would add a heading to define this as a specific section, maybe something like "Supported issuer types for Knative Serving encryption features"